devtools/mkgossip.c

```diff
@@ -298,9 +311,13 @@ int main(int argc, char *argv[])
 
 	printf("type=channel_announcement\n");
 	printf("   node_signature_1=%s\n", sig_as_hex(&nodesig[lesser_key]));
+	print_hash_and_privkey(&node_privkey[0], &hash);
 	printf("   node_signature_2=%s\n", sig_as_hex(&nodesig[!lesser_key]));
+	print_hash_and_privkey(&node_privkey[1], &hash);
 	printf("   bitcoin_signature_1=%s\n", sig_as_hex(&bitcoinsig[lesser_key]));
+	print_hash_and_privkey(&funding_privkey[0], &hash);
 	printf("   bitcoin_signature_2=%s\n", sig_as_hex(&bitcoinsig[!lesser_key]));
+	print_hash_and_privkey(&funding_privkey[1], &hash);
```

Comment:
Same as above, so it can be cut & paste.  Maybe print a real signature in hex with --verbose or something?